src/extension/tsconfigHelper.ts

```diff
@@ -23,24 +24,27 @@ export class TsConfigHelper {
         return fileSystem.exists(tsConfigPath)
             .then(function(exists: boolean): Q.Promise<void> {
                 if (!exists) {
-                    return fileSystem.writeFile(tsConfigPath, "{}");
+                    return fileSystem.writeFile(tsConfigPath, "{ compilerOptions: { allowJs: true } }");
```

Comment:
I meant the whole thing though. With the code as you have it here, the intellisense helper class (which is the only caller of this `TsConfigHelper` class) will still always modify the tsconfig file, which can potentially trample the user's settings and runs into issues with comments. With the changes I propose, we would only ever create the file completely, never modify a pre-existing file, which means we don't need to worry about parsing it, and we don't trample user settings.